project/Deliverable1.md

```diff
@@ -274,7 +274,7 @@ The API is comprised of three interfaces. You **must not** change the interface
 
 - `InsightResponse` is the interface for the objects your methods will fulfill with.
 - `IInsightFacade` is the front end (wrapper) for the query engine. In practice, it defines the endpoints for the deliverable. It provides several methods:
-- `addDataset(id: string, content: string): Promise<InsightResponse>` adds a dataset to the internal model, providing the id of the dataset, and the string of the content of the dataset.
+- `addDataset(id: string, content: string, type: InsightDatasetType): Promise<InsightResponse>` adds a dataset to the internal model, providing the id of the dataset, the string of the content of the dataset, and the type of the dataset. For this deliverable the dataset type will be _courses_.
```

Comment:
This will end up being `IInsightDatasetKind`, and the param should be `kind`. Also, there should be the `listDatasets` method too.